index.bs

```diff
@@ -1994,93 +2098,203 @@ The [=remote end steps=] with |command parameters| are:
 
   1. Let |request| be a new [=/request=] whose URL is |url record|.
 
-  1. Let |navigation id| be the string representation of a
-     [[!RFC4122|UUID]] based on truly random, or pseudo-random numbers.
+  1. Return the result of [=await a navigation=] with |context| and |request|.
 
-  1. [=Navigate=] |context| with resource |request|, and using |context| as the
-     [=source browsing context=], and with navigation id |navigation id|.
+</div>
 
-  1.  Let (|event received|, |navigate status|) be [=await=] given
-      «"<code>navigation started</code>", "<code>navigation failed</code>",
-      and "<code>fragment navigated</code>"» and |navigation id|.
+#### The browsingContext.reload Command ####  {#command-browsingContext-reload}
 
-  1. Assert: |navigate status|'s id is |navigation id|.
+The <dfn export for=commands>browsingContext.reload</dfn> command reloads a
+browsing context.
 
-  1. If |navigate status|'s status is "<code>complete</code>":
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      BrowsingContextReloadCommand = {
+        method: "browsingContext.reload",
+        params: BrowsingContextReloadParameters
+      }
 
-    1. Let |body| be a [=map=] matching the
-       <code>BrowsingContextNavigateResult</code> production, with the
-       <code>navigation</code> field set to |navigation id|, and the
-       <code>url</code> field set to the result of the [=URL serializer=] given
-       |navigate status|'s url.
+      BrowsingContextReloadParameters = {
+        context: BrowsingContext,
+        ?ignoreCache: boolean,
+        ?wait: ReadinessState,
+      }
+      </pre>
+   </dd>
+</dl>
 
-    1. Return [=success=] with data |body|, and then run the following steps [=in
-       parallel=]:
+<div algorithm="remote end steps for browsingContext.reload">
+The [=remote end steps=] with |command parameters| are:
 
-       1. Run the [=WebDriver-BiDi fragment navigated=] steps given |context|
-          and |navigate status|
+1. Let |context id| be the value of the <code>context</code> field of
+   |command parameters|.
 
-     Note: this is the case if the navigation only caused the fragment to
-     change. The parallel steps here ensure that we return the command result
-     before emitting the event, so the navigation id is known.
+1. Let |context| be the result of [=trying=] to [=get a browsing context=]
+   with |context id|.
 
-  1. If |navigate status|'s status is "<code>canceled</code>" return [=error=]
-     with [=error code=] [=unknown error=].
+1. Assert: |context| is not null.
 
-     TODO: is this the right way to handle errors here?
+1. Let |ignore cache| be the the value of the <code>ignoreCache</code> field of |command
+   parameters| if present, or false otherwise.
 
```

Comment:
```suggestion
1. Let |wait condition| be the value of the <code>wait</code> field of |command
   parameters| if present, or "<code>none</code>" otherwise.
```

Reply:
Also need to pass this to `await a navigation` below.